types/pg/index.d.ts

```diff
@@ -18,6 +18,7 @@ export interface ConnectionConfig {
     connectionString?: string;
     keepAlive?: boolean;
     stream?: stream.Duplex;
+    statement_timeout?: false | number;
```

Comment:
Why is `false` and option here?

Reply:
'statement_timeout' default is 'false', meaning 'unlimited'; see here: 

- https://github.com/brianc/node-postgres/blob/1cbd507b8ca0416b9be0252cc930762e716ba60f/lib/connection-parameters.js#L67
- https://github.com/brianc/node-postgres/blob/1cbd507b8ca0416b9be0252cc930762e716ba60f/lib/defaults.js#L58

'true' has no meaning, and number is anything which is not 'unlimited'.